api/src/main/java/me/lokka30/treasury/api/economy/response/EconomyFailureReason.java

```diff
@@ -59,29 +68,41 @@
         }
     },
 
-    /* Accounts */
-
     /**
      * A constant representing failure due to the inability to locate an
      * {@link me.lokka30.treasury.api.economy.account.Account Account}.
+     * To avoid this failure, check if the account exists before attempting to
+     * interact with it.
      *
+     * @see EconomyProvider#hasAccount(String, EconomySubscriber)
+     * @see EconomyProvider#hasPlayerAccount(UUID, EconomySubscriber)
+     * @see EconomyProvider#retrieveAccount(String, EconomySubscriber)
+     * @see EconomyProvider#retrievePlayerAccount(UUID, EconomySubscriber)
      * @since {@link me.lokka30.treasury.api.economy.misc.EconomyAPIVersion#v1_0 v1.0}
      */
     ACCOUNT_NOT_FOUND {
         /**
          * {@inheritDoc}
          */
         @Override
-        public @NotNull String getDescription() {
+        @NotNull
```

Comment:
May want to inline the annotation here again to be consistent with the rest of the class.